sig/http/response.rbs

```diff
@@ -0,0 +1,26 @@
+module Algolia
+  module Http
+    class Response
+      attr_reader status: untyped
+
+      attr_reader body: untyped
+
+      attr_reader error: untyped
+
+      attr_reader headers: untyped
+
+      attr_reader has_timed_out: bool
+
+      attr_reader network_failure: untyped
+
+      #
+      # @option status    [String]  Response status
+      # @option body    [String]  Response body
+      # @option error    [String]  Response error or caught error
+      # @option headers    [String]  Response headers
+      # @option has_timed_out    [String]  If the request has timed out
+      #
+      def initialize: (?::Hash[Symbol, String | bool] opts) -> untyped
```

Comment:
For readability I'd be in favor of deleting the spaces around the `|`, just for the RBS definition. I find that overall easier to read. What are your thoughts? I'd also be in favor of removing the dockblocks are they're not useful in `.rbs` files, and deleting the extra lines between classes variables etc (leaving only one between the variables and the methods)

Reply:
Yes, agreed!